Performance/.gitignore

```diff
@@ -0,0 +1,4 @@
+Generated/
+Results/
+Sources/*+Generated.swift
```

Comment:
Will generating things in here cause problems if they are laying around when someone builds in general?